models/base_model.py

```diff
@@ -58,7 +51,7 @@ def save(self):
         models.storage.new(self)
         models.storage.save()
 
-    def to_dict(self):
+    def to_dict(self, secure_pwd=True):
```

Comment:
Why was the parameters of this function updated?

Reply:
that are requirements for task 15 , is not finished